samples/Kubernetes/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+# Tell us about your experience using dotnet monitor: https://aka.ms/dotnet-monitor-survey
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deploy-exampleapp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: exampleapp
+  template:
+    metadata:
+      labels:
+        app: exampleapp
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: app
+        image: mcr.microsoft.com/dotnet/samples:aspnetapp-chiseled
+        imagePullPolicy: Always
+        env:
+        - name: DOTNET_DiagnosticPorts
+          value: /diag/dotnet-monitor.sock
+        volumeMounts:
+        - mountPath: /diag
+          name: diagvol
+        resources:
+          limits:
+            cpu: 250m
+            memory: 512Mi
+      - name: monitor
+        image: mcr.microsoft.com/dotnet/monitor:8
+        # DO NOT use the --no-auth argument for deployments in production; this argument is used for demonstration
```

Comment:
Just a bit of a quirk with some of our comments in a couple of places - if someone references this yaml in isolation, these don't mean much and don't tell you where to get clarification.

Reply:
I don't think many folks will find this file in isolation. They are likely to get to it from the kubernetes and docker markdown files. It might be good to have readme markdowns for each sample that explain the nuances right next to these files. I'll think about it and propose a separate change for this.